### ISSUE TITLE
How to change the b_cap slave mode in ROS for cobotta

### ISSUE THREAD

**Issue by zhoubo2018** [DENSORobot/denso_robot_ros#20]
Hello, after some testing the cobotta ros driver, we found sometimes the arm moveit will unexpected stop when using moveit (in remote app, there is some errors says "Slvmove instruction is executed in non-subbureau mode" and "instruction delay is generated"). the denso support ask me to change the b_cap slave mode in ros driver, so could you tell me where to change this parameter in this ros package? thanks a lot.

**Comment by zhoubo2018** [comment]
I mean to change b_cap slave mode 0 to 1, just like that.

**Comment by ToshitakaSuzuki** [comment]
Sorry for my late reply.

You can change slave mode by publishing the following topic:
 `rostopic pub /cobotta/ChangeMode std_msgs/Int32 "data: 2" `
"data:2" means changing to mode 2.

**Comment by zhoubo2018** [comment]
yes, I have tried that, but it will first occur an error"Failed to write", and cobotta arm LED will become green, then I tried to type again, it will no error then the cobotta LED turn to white, is that right? Do I successfully change the mode?
